SECURITY.md

```diff
@@ -0,0 +1,10 @@
+Security is taken very seriously here, therefore we follow the rule of responsible
+disclosure and we encourage you to do so.
```

Comment:
```suggestion
Authelia takes security very seriously. We follow the rule of [responsible disclosure](https://en.wikipedia.org/wiki/Responsible_disclosure), and we encourage the community to as well.
```

Thoughts?

Reply:
yeah, I did not properly rebase I guess.

Reply:
I aligned all documents in the next commit. Thanks.